### PR TITLE
Updating Carl's caption

### PR DESCRIPTION
**Pull request by Carl-VT88**
Merge the carl branch into Main

### CODE REVIEW

File: _slides/04-Carl-VT88.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+This is a picture of Carl
```
Comment (ejchung11) [inline _slides/04-Carl-VT88.md:6]: Looks ok!

Comment (ejchung11) [reply _slides/04-Carl-VT88.md:6]: Good

**Review by ejchung11 — APPROVED** [review]
ok

**Comment by Carl-VT88** [comment]
comment added to pull merging

**Pull request merged**